docs/docs/slot-validation-actions.mdx

```diff
@@ -7,6 +7,12 @@ abstract: A slot validation action is a special type of custom action, designed
 
 You can do custom extraction and validation of slot values by writing custom actions in three different ways.
 
+:::note Extraction and Validation can work together
+
+You can have an `Action` and a `ValidationAction` separately and both of these actions will be called from Rasa. For example, you can use a normal `Action` to create several events and a `ValidationAction` to validate that slot values are correct. In this scenario, `Action` will be called first and `ValidationAction` afterwards.
+
+:::
```

Comment:
Is this no longer true?
> A slot validation action should only return `slot` and [`bot`](https://rasa.com/docs/action-server/events#bot) events.
Any other event type will be filtered out by the default action [`action_extract_slots`](./default-actions.mdx#action_extract_slots).

Reply:
In that case this needs some rephrasing, and I'd suggest moving it down to the end because at the beginning of the doc the user hasn't read about ValidationAction yet.

```suggestion
:::note Using different actions for extraction and validation

You can use both a regular custom action and `action_validate_slot_mappings` together to extract and validate a slot. For example, you can specify a regular custom action as the `action` for a `custom` slot mapping, and also add validation logic for the same slot to `action_validate_slot_mappings`. The custom action specified in the custom slot mapping will be called first, and `action_validate_slot_mappings` will be called afterwards.

:::
```

Reply:
This is much better wording, thanks @melindaloubser1